dw-action-toolbar.js

```diff
@@ -398,44 +402,41 @@ export class DwActionToolbar extends LitElement {
    */
   _removeHiddenActions(aActions) {
     let actions = [...aActions];
-    if(isEmpty(this.hiddenActions)) {
+    if (isEmpty(this.hiddenActions)) {
       return actions;
     }
 
     let result = [];
     actions.forEach((action) => {
-      if(this.hiddenActions.indexOf(action.name) === -1) {
-        if (action.type === 'expandable' && action.subActions && action.subActions.length) {
+      if (this.hiddenActions.indexOf(action.name) === -1) {
+        if (action.type === "expandable" && action.subActions && action.subActions.length) {
           const subActions = [];
           action.subActions.forEach((action) => {
             if (this.hiddenActions.indexOf(action.name) === -1) {
               subActions.push(action);
             }
           });
-          action= {...action, subActions};
+          action = { ...action, subActions };
         }
         result.push(action);
       }
-
     });
     return result;
   }
 
   /**
-   * Open action toolbar.
+   * Open Menu
    */
-  open() {
-    let elSelect = this.shadowRoot.querySelector('dw-select');
-    elSelect && elSelect.open();
+  _onMenuOpen() {
```

Comment:
This handler function isn't named properly. It's click handler of `vert-more`. so, it can be better named as `_onVertMmoreClick`